cmd/goal/account.go

```diff
@@ -613,12 +618,12 @@ var addParticipationKeyCmd = &cobra.Command{
 var installParticipationKeyCmd = &cobra.Command{
 	Use:   "installpartkey",
 	Short: "Install a participation key",
-	Long:  `Install a participation key`,
+	Long:  `Install a participation key from a partkey file. Intended for use with participation key files generated by "algokey part generate". Does not change the online status of an account or register the participation key; use "goal account changeonlinestatus" for doing so.`,
```

Comment:
We need to encourage proper partkey security.  Should we delete the original partkey file if the installPartKey succeeds?  If not, we should print out a message afterward to tell them they should delete it (or otherwise secure it).

Reply:
Yeah, you're probably right.  I initially wasn't going to do this because deleting keys seemed risky, but in the case of install, sqlite guarantees durability for the key we just installed into the data directory, so it should be safe to delete because we know a copy is durably stored with algod now.  Pushed code to do that.